examples/main/src/layer-examples.js

```diff
@@ -209,6 +210,26 @@ const GridLayerExample = {
   }
 };
 
+const PointDensityGridLayerExample = {
+  layer: PointDensityGridLayer,
+  props: {
+    id: 'pointDensityGridLayer',
+    data: dataSamples.points,
+    extruded: true,
+    pickable: true,
+    opacity: 1,
+    cellSize: 0.3,
+    lightSettings: {
```

Comment:
For now, should there be one global lightSettings object in this file, so that all layers in the example get the same lighting?

@pessimistress For longer term, I have suggested that we define a deck.gl `Effect` to contain the global lighting props, then we could have a single control panel for the global lighting props just like we have a control panels for all layer props. Will provide some more detail on this idea separately.